gcloud/_helpers.py

```diff
@@ -238,3 +241,35 @@ def __init__(self, project=None, implicit=False):
 
 
 _DEFAULTS = _DefaultsContainer(implicit=True)
+
+
+class _ClientProxy(object):
```

Comment:
There really isn't much "there" implementing the wrappers for `Topic` and `Subscription`.  Without the wrappers, the "real" `Topic` and `Subscription` classes still have to have a `connection` passed to their API-driving methods, which defeats the goal of clients.

Or else we have to weld in the "client" as an instance attribute of the "real" `Topic` and `Subscription`.

There really doesn't seem to be any other alternative (I've been looking for one for a couple of weeks now).